credentialsdemo/source/no-sdk/src/org/kaaproject/kaa/examples/credentials/CredentialsDemo.java

```diff
@@ -0,0 +1,97 @@
+/**
+ *  Copyright 2014-2016 CyberVision, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.kaaproject.kaa.examples.credentials;
+
+import org.kaaproject.kaa.examples.credentials.kaa.KaaAdminManager;
+import org.kaaproject.kaa.examples.credentials.utils.CredentialsConstants;
+import org.kaaproject.kaa.examples.credentials.utils.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A demo application that shows how to use the Kaa credentials API.
+ *
+ * @author Maksym Liashenko
+ */
+public class CredentialsDemo {
+    private static final Logger LOG = LoggerFactory.getLogger(CredentialsDemo.class);
+
+    private static void getTenantAdminCredentials() {
+        LOG.info("Please, input your tenant admin username (push <Enter> if want to use default): ");
+        String username = IOUtils.getUserInput();
+        if (!"".equals(username)) {
+            CredentialsConstants.TENANT_ADMIN_USERNAME = username;
+        }
+        LOG.info("Please, input your tenant admin password (push <Enter> if want to use default): ");
+        String pass = IOUtils.getUserInput();
+        if (!"".equals(pass)) {
+            CredentialsConstants.TENANT_ADMIN_USERNAME = pass;
+        }
+    }
+
+    public static void main(String[] args) throws InterruptedException {
+        LOG.info("Credentials demo: admin part started!");
+
+        LOG.info("We need more info about your system. Please provide it.");
+        getSandboxIp();
+        getTenantAdminCredentials();
```

Comment:
I propose to pass `sandbox host/ip` parameter as program cli argument (see "activation demo") and as credentials use `devuser/devuser123` or `admin/admin123` depending on required action.